### ISSUE TITLE
Make App Store icon work with VoiceOver

### ISSUE THREAD

**Issue by mi-wood** [RefugeRestrooms/refugerestrooms#384]
@DeeDeeG : 

> The App Store download button isn't focusable with VoiceOver, so it would be nice to make that focusable. Not sure how at the moment, but should be doable.

**Comment by stardust66** [comment]
What do you mean by this?
<img width="1232" alt="screen shot 2017-11-17 at 6 25 31 pm" src="https://user-images.githubusercontent.com/11802391/32973322-ccf4ad8a-cbc4-11e7-8b98-5ea7048050c8.png">
Voiceover on Mac with Google Chrome seems to be able to focus on the button.

**Comment by DeeDeeG** [comment]
It may be a quirk with the software on iOS 9, because iPad 2 (the model I have) only updates to iOS 9.3.5.

I can give it a quick try again.

Also might be specific to touch screens?

I was thinking of attempting to fix this by giving the App Store button an HTML attribute [`tabindex=0`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) (and seeing if that fixes it).

Anyways, I'm going to go check this with the iPad again and see if it's still not focusing.

**Comment by DeeDeeG** [comment]
I went and confirmed it again.

When visiting the site (as hosted at refugerestrooms.org as of Nov 17 2017) on an iPad 2 or iPad 3, with iOS 9.3.5 installed, the App Store button can't be focused with VoiceOver.

I don't have any bluetooth keyboards or anything, I'm just using the touchscreen.

**Comment by DeeDeeG** [comment]
I knew I read somewhere that this was a Safari/Webkit bug. Here's the very bug report itself: https://bugs.webkit.org/show_bug.cgi?id=145263

Old versions of Safari used with VoiceOver totally skip SVGs in \<img> elements. Fixed in either iOS 10 or iOS 11. My old iPad 2 and iPad 3 are still affected, along with anyone else using the same models or earlier. Thankfully, newer models aren't affected, but it's still an issue for some folks out there.
